src/yunohost/tools.py

```diff
@@ -565,3 +565,25 @@ def tools_diagnosis(auth, private=False):
         diagnosis['private']['domains'] = domain_list(auth)['domains']
 
     return diagnosis
+
+def tools_public_key():
+    key_path = '/root/.ssh/id_rsa_ynh'
```

Comment:
as `admin` user is used for many administration tasks, should we rather create the key in `/home/admin` ?